resources/js/Shared/Modules/FeedItems/MoodTrackingEvent.vue

```diff
@@ -0,0 +1,55 @@
+<template>
+  <div class="rounded-lg border border-gray-300 dark:border-gray-700">
+    <!-- contact information -->
+    <div
+      v-if="!contactViewMode"
+      class="flex items-center border-b border-gray-300 px-3 py-2 text-sm dark:border-gray-700">
+      <avatar
+        :data="data.contact.avatar"
+        :classes="'rounded-full border-gray-200 dark:border-gray-800 border relative h-5 w-5 mr-2'" />
+
+      <div class="flex flex-col">
+        <inertia-link :href="data.contact.url" class="text-gray-800 hover:underline dark:text-gray-200">{{
+          data.contact.name
+        }}</inertia-link>
+      </div>
+    </div>
+
+    <div class="px-3 py-2">
+      <!-- the mood tracking event still exists in the system -->
+      <div v-if="data.mood_tracking_event.object" class="flex items-center">
+        <span class="mr-2">{{ data.mood_tracking_event.description }}</span>
+        <span class="mr-2"><span class="text-gray-500">for</span> {{ data.mood_tracking_event.object.rated_at }}</span>
+        <span v-if="data.mood_tracking_event.object.number_of_hours_slept"
+          >and slept {{ data.mood_tracking_event.object.number_of_hours_slept }} hours</span
+        >
```

Comment:
This will be too complicated to translate.

Reply:
Very. I agree.

Reply:
fixed